socs/agents/synacc/agent.py

```diff
@@ -17,14 +17,22 @@ class SynaccessAgent:
         password(str): Password credential to login to device.
     """
 
-    def __init__(self, agent, ip_address, username, password):
+    def __init__(self, agent, ip_address, username, password, outlet_names=None):
         self.agent = agent
         self.log = agent.log
         self.lock = TimeoutLock()
         self.ip_address = ip_address
         self.user = username
         self.passw = password
 
+        if outlet_names is None:
+            self.outlet_names = ['outlet1', 'outlet2', 'outlet3', 'outlet4', 'outlet5']
+        else:
+            for i in range(5):
```

Comment:
Are all synaccess PDUs 5 outlets? (I suspect @mhasself was suggesting a constant be used that could be defined elsewhere, in the event that the outlet count differs.)

I think they do make different size units, but maybe all the ones we use are 5 outlets?